website/docs/components/alert/partials/guidelines/guidelines.md

```diff
@@ -29,86 +29,86 @@ Use compact alerts when a less prominent alert is needed.
 ## Color
 
 <Doc::Layout @spacing="12px">
-  <Hds::Alert @type="inline" @color="neutral" as |A|>
-    <A.Title>Neutral alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
-  <Hds::Alert @type="inline" @color="highlight" as |A|>
-    <A.Title>Highlight alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
-  <Hds::Alert @type="inline" @color="success" as |A|>
-    <A.Title>Success alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
-  <Hds::Alert @type="inline" @color="warning" as |A|>
-    <A.Title>Warning alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
-  <Hds::Alert @type="inline" @color="critical" as |A|>
-    <A.Title>Critical alert title</A.Title>
-    <A.Description>Lorem ipsum dolar sit amet.</A.Description>
-  </Hds::Alert>
+<Hds::Alert @type="inline" @color="neutral" as |A|>
+<A.Title>Neutral alert title</A.Title>
+<A.Description>Lorem ipsum dolar sit amet.</A.Description>
+</Hds::Alert>
+<Hds::Alert @type="inline" @color="highlight" as |A|>
+<A.Title>Highlight alert title</A.Title>
+<A.Description>Lorem ipsum dolar sit amet.</A.Description>
+</Hds::Alert>
+<Hds::Alert @type="inline" @color="success" as |A|>
+<A.Title>Success alert title</A.Title>
+<A.Description>Lorem ipsum dolar sit amet.</A.Description>
+</Hds::Alert>
+<Hds::Alert @type="inline" @color="warning" as |A|>
+<A.Title>Warning alert title</A.Title>
+<A.Description>Lorem ipsum dolar sit amet.</A.Description>
+</Hds::Alert>
+<Hds::Alert @type="inline" @color="critical" as |A|>
+<A.Title>Critical alert title</A.Title>
+<A.Description>Lorem ipsum dolar sit amet.</A.Description>
+</Hds::Alert>
```

Comment:
Curious why it wipes out the indenting?

Reply:
Whoops, sometimes my Prettier config does this, I'll revert that!